### PR TITLE
Eliminate unnecessary errors in the log

### PR DESCRIPTION
**Pull request by keliramu**

### CODE REVIEW

File: norduser/service/fork.go [hunk 1]
```diff
@@ -76,15 +81,16 @@ func findPIDOfUID(uids string, desiredUID uint32) int {
 		var pid int
 		var uid int
 		n, err := fmt.Sscanf(uidPid, "%d%d", &uid, &pid)
+		if errors.Is(err, io.EOF) {
```
Comment (keliramu) [inline norduser/service/fork.go:84]: There were many of these `[Error] failed to parse uid pid line:  EOF`

File: norduser/service/fork.go [hunk 2]
```diff
@@ -49,9 +50,13 @@ func handlePsError(out []byte, err error) error {
 func parseNorduserPIDs(psOutput string) []int {
 	pids := []int{}
 	for _, pidStr := range strings.Split(psOutput, "\n") {
-		pid, err := strconv.Atoi(strings.TrimSpace(pidStr))
+		pidStr = strings.TrimSpace(pidStr)
+		if pidStr == "" {
```
Comment (keliramu) [inline norduser/service/fork.go:54]: There were many of these `failed to parse pid string:  strconv.Atoi: parsing "": invalid syntax`

File: norduser/service/fork.go [hunk 3]
```diff
@@ -76,15 +81,16 @@ func findPIDOfUID(uids string, desiredUID uint32) int {
 		var pid int
 		var uid int
 		n, err := fmt.Sscanf(uidPid, "%d%d", &uid, &pid)
+		if errors.Is(err, io.EOF) {
+			break
```
[the remark this answers is not in the document]
Comment (keliramu) [reply norduser/service/fork.go:85]: This error means it's over - nothing more to parse, break the loop and return -1, invoking fn check this case as process is not running and nothing to stop or restart.

File: norduser/service/fork.go [hunk 4]
```diff
@@ -49,9 +50,13 @@ func handlePsError(out []byte, err error) error {
...
 	for _, pidStr := range strings.Split(psOutput, "\n") {
-		pid, err := strconv.Atoi(strings.TrimSpace(pidStr))
+		pidStr = strings.TrimSpace(pidStr)
+		if pidStr == "" {
+			continue
+		}
+		pid, err := strconv.Atoi(pidStr)
 		if err != nil {
-			log.Println("failed to parse pid string: ", err)
+			log.Println(internal.ErrorPrefix+" failed to parse pid string:", err)
```
[the remark this answers is not in the document]
Comment (keliramu) [reply norduser/service/fork.go:59]: Had temptation to change that way but left original version. IMO, it should be that way as you say, but other dev mix these approaches...

**Review by mariusSincovici — APPROVED** [review]

**Review by Savolro — APPROVED** [review]

**Pull request merged**